scripts/dist-tag.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+// tslint:disable: no-implicit-dependencies
+
+import { logging } from '@angular-devkit/core';
+import { execSync } from 'child_process';
+import { packages, stableToExperimentalVersion } from '../lib/packages';
+
+interface DistTagOptions {
+  /**
+   * Version must be specified in format d+.d+.d+ where d is a 0-9 digit.
```

Comment:
Nit: Should give an example here so humans don't need to parse regexes. Maybe include:

> For example, `1.2.3` or `10.0.0`.

Is there any scenario where we would want to use `-next` or `-rc`? I'm guessing not, but we should explicitly state one way or the other in the doc.

Reply:
Mentioned both -next and -rc.